libs/full/naming/include/hpx/modules/naming.hpp

```diff
@@ -0,0 +1,10 @@
+//  Copyright (c) 2020 Hartmut Kaiser
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/config.hpp>
+#include <hpx/modules/naming_base.hpp>
```

Comment:
Is this file just preparing for further changes to the `naming` module (since it doesn't look very useful at the moment...)?

Reply:
Yes, it's a leftover, which I hope will be more significant in the future. I have moved everything into the `naming_base` module while working on this PR and didn't want to remove the now empty module as I'm sure we will need it.